base/settings/R/pft_site_linker.R

```diff
@@ -0,0 +1,66 @@
+#'
+#' @title site.pft.linkage
+#' 
+#' @param pecan.xml.address pecan settings list.
```

Comment:
this doesn't match the function itself, where this is just `settings`
```suggestion
#' @param settings pecan settings list.
```